### PR TITLE
Update netlify.mdx

### PR DESCRIPTION
**Pull request by alexanderniebuhr**
#### Description (required)

related https://github.com/withastro/adapters/pull/120

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by alexanderniebuhr** [comment]
cc @Skn0tt

File: src/content/docs/en/guides/integrations-guide/netlify.mdx [hunk 1]
```diff
@@ -131,7 +131,7 @@ Any Astro middleware is applied to pre-rendered pages at build-time, and to on-d
 
 To implement redirects, access control or custom response headers for pre-rendered pages, run your middleware on Netlify Edge Functions by enabling the [`edgeMiddleware` option](/en/reference/adapter-reference/#edgemiddleware):
 
-```js title="astro.config.mjs" ins={7}
+```js title="astro.config.mjs" ins={8}
```
Comment (alexanderniebuhr) [inline src/content/docs/en/guides/integrations-guide/netlify.mdx:134]: Unrelated to the related change, but the highlight was off by one line

File: src/content/docs/en/guides/integrations-guide/netlify.mdx [hunk 2]
```diff
@@ -148,9 +148,24 @@ Configuring `edgeMiddleware: true` will deploy your middleware as an Edge Functi
 
 ### Netlify Image CDN support
 
-This adapter uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
+This adapter by default uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
 It's implemented using an [Astro Image Service](/en/reference/image-service-reference/) under the hood.
 
+To opt out of Netlify's Image CDN remote image optimization use the `imageCDN` option:
```
Comment (sarah11918) [inline src/content/docs/en/guides/integrations-guide/netlify.mdx:154]:
```suggestion
To opt out of Netlify's Image CDN remote image optimization, use the `imageCDN` option:
```

**Review by sarah11918 — APPROVED** [review]
One little comma missing, but otherwise looks great! Since the feature has already been released, feel free to merge at will.

File: src/content/docs/en/guides/integrations-guide/netlify.mdx [hunk 3]
```diff
@@ -148,9 +148,24 @@ Configuring `edgeMiddleware: true` will deploy your middleware as an Edge Functi
 
 ### Netlify Image CDN support
 
-This adapter uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
+This adapter by default uses the [Netlify Image CDN](https://docs.netlify.com/image-cdn/) to transform images on-the-fly without impacting build times.
```
Comment (andrewdisley) [inline src/content/docs/en/guides/integrations-guide/netlify.mdx:151]: The URL https://docs.netlify.com/image-cdn/ 404’s - it should be https://docs.netlify.com/image-cdn/overview/

Was going to open a PR for it but might as well catch it here if you can @alexanderniebuhr

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Skn0tt — COMMENTED** [review]
Looks good 👍 Thanks Sarah!

**Pull request merged**